src/Client/Dockerfile

```
@@ -0,0 +1,43 @@
+# 🌍 Set environment variables
+ENV NODE_ENV=production
+ENV PORT=9999
+ENV HOSTNAME="0.0.0.0"
+
+# 📦 Copy build artifacts to reduce image size
+COPY --from=builder --chown=nextjs:nodejs /app/.next/standalone ./
+COPY --from=builder --chown=nextjs:nodejs /app/.next/static ./.next/static
+
+# 🔐 Create a non-root user and group
+RUN addgroup --system --gid 1001 nodejs
+RUN adduser --system --uid 1001 nextjs
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)